site/content/docs/03-run-time.md

```diff
@@ -214,7 +214,11 @@ Events dispatched from child components can be listened to in their parent. Any
 
 ### `svelte/store`
 
-The `svelte/store` module exports functions for creating [stores](docs#4_Prefix_stores_with_$_to_access_their_values).
+The `svelte/store` module exports functions for creating [readable](docs#readable), [writable](docs#writable) and [derived](docs#derived) stores.
+
+Keep in mind that you don't *have* to use these functions to enjoy the [reactive `$store` syntax](docs#4_Prefix_stores_with_$_to_access_their_values) in your components. Any object that correctly implements `subscribe`/`unsubscribe` and (optionally) `set` is a valid store, and will work both with the special syntax, and with Svelte's built in [`derived` stores](docs#derived).
```

Comment:
Nit: 'built-in' is how we're writing this elsewhere in the docs.

Also, writing '`subscribe`/`unsubscribe`' seems a little confusing, since `unsubscribe` isn't actually the name of any method. I'm not sure the best way to convey this. '`subscribe`/unsubscribe' looks a bit weird.